modules/twinkleprotect.js

```diff
@@ -993,6 +1065,70 @@ Twinkle.protect.protectionTags = [
 // Filter FlaggedRevs
 .filter((type) => hasFlaggedRevs || type.label !== 'Pending changes templates');
 
+Twinkle.protect.ctopCodes = [
+	{ value: '', label: 'Specify a code...' },
+	{ value: 'A-A', label: 'Armenia-Azerbaijan' },
+	{ value: 'AB', label: 'Abortion' },
+	{ value: 'A-I', label: 'Palestine-Israel articles', aliases: ['PIA'] },
+	{ value: 'AP', label: 'American politics' },
+	{ value: 'APL', label: 'Antisemitism in Poland' },
+	{ value: 'AT', label: 'Article titles and capitalisation' },
+	{ value: 'BLP', label: 'Editing of Biographies of Living Persons' },
+	{ value: 'CAM', label: 'Acupuncture' },
+	{ value: 'CC', label: 'Climate change' },
+	{ value: 'CF', label: 'Pseudoscience' },
+	{ value: 'CID', label: 'Civility in infobox discussions' },
+	{ value: 'COVID', label: 'COVID-19' },
+	{ value: 'EE', label: 'Eastern Europe' },
+	{ value: 'FG', label: 'Falun Gong' },
+	{ value: 'GC', label: 'Gun control' },
+	{ value: 'GG', label: 'Gender and sexuality' },
+	{ value: 'GMO', label: 'Genetically modified organisms' },
+	{ value: 'HORN', label: 'Horn of Africa' },
+	{ value: 'IMH', label: 'Indian military history' },
+	{ value: 'IRP', label: 'Iranian politics' },
+	{ value: 'KURD', label: 'Kurds and Kurdistan' },
+	{ value: 'RI', label: 'Race and intelligence' },
+	{ value: 'RNE', label: 'Historical elections' },
+	{ value: 'SA', label: 'South Asia' },
+	{ value: 'SASG', label: 'South Asian social groups', aliases: ['GSCASTE'] },
+	{ value: 'TT', label: 'The Troubles' },
+	{ value: 'YA', label: 'Yasuke' }
+];
+
+// We add the code at the front of each label
+Twinkle.protect.ctopCodes.forEach((item) => {
+	if (item.value) {
+		if (item.aliases) {
+			item.label = `${item.value}, ${item.aliases.join(', ')} – ${item.label}`;
+		} else {
+			item.label = `${item.value} – ${item.label}`;
+		}
+	}
+});
+
+Twinkle.protect.gsCodes = [
+	{ value: '', label: 'Specify a code...' },
```

Comment:
Same as above comment. Let's pull these from on-wiki if we can. Maybe create a new .json page at https://en.wikipedia.org/wiki/Template:Gs/topics.json ?